db/migrate/20230916124819_add_hmis_view_project_permission.rb

```diff
@@ -0,0 +1,10 @@
+class AddHmisViewProjectPermission < ActiveRecord::Migration[6.1]
+  def up
+    Hmis::Role.ensure_permissions_exist
+    Hmis::Role.reset_column_information
+  end
+
+  def down
+    remove_column :hmis_roles, :can_view_project
```

Comment:
Random thought, but I wonder if this is more like `can_interact_with_associated_projects_at_ALL`.

Reply:
Hm, maybe.
Also/relatedly the description for "Can view enrollment details" and "can edit enrollments" maybe updated to explain that "can view projects" must be present for them to apply. Or, thats like a general comment about anything in the "Enrollments" and "Projects" category. Not the "Client Access" category, though.